.circleci/config.yml

```diff
@@ -120,7 +120,7 @@ jobs:
     environment:
     - CIRCLECI_CLI_VERSION: 0.1.5546
     - GO_TAGS: ''
-    - GO_VERSION: 1.17.11
+    - GO_VERSION: 1.18.3
```

Comment:
Out of curiosity, is there a change in this PR that requires the updated Go version?

Updating Go versions is a little more involved than this (see some past PRs for example) and is usually something we do closer to release time.

Reply:
This would cause the tests to fail due to the SHA-1 deprecation, so even more reason to move to a separate repo. :)